app/controllers/user_controller.rb

```diff
@@ -125,23 +125,24 @@ def disable_api_key
   end
 
   def profile
-    #find the user if it isn't already set
-    unless @user
-      @user = User.friendly.find(params[:id])
-    end
+    # Find the user if it isn't already set
+    @user ||= User.friendly.find(params[:id])
+
     if !@user.deleted || current_user.admin
       @collections_and_document_sets = @user.visible_collections_and_document_sets(current_user)
-      @collection_ids = @collections_and_document_sets.map {|collection| collection.id}
-      @deeds = @user.deeds.includes(:note, :page, :user, :work, :collection).order('created_at DESC').paginate :page => params[:page], :per_page => PAGES_PER_SCREEN
+      @collection_ids = @collections_and_document_sets.map(&:id)
```

Comment:
This is interesting -- sometime we should chat about `&:` , `lambda`, and `->` in the dev channel, as these are newish features of the language which @saracarl and I are weak on.